### PR TITLE
Feature/hrt 156 household members

### PR DESCRIPTION
**Pull request by tony-gledhill**
[HRT-156](https://hackney.atlassian.net/browse/HRT-156?atlOrigin=eyJpIjoiMWI2ZDAzNGIwMWVhNGQ2NjhjZDUyODVlMmVjOGY3MWEiLCJwIjoiaiJ9)

This change allows for admin users to view the details of applicants via the admin screens.

### CODE REVIEW

File: pages/applications/view/[id]/[person]/index.tsx [hunk 1]
```diff
@@ -41,40 +51,91 @@ export default function ApplicationPersonPage({
     ? data.mainApplicant
     : data.otherMembers?.find((x) => x.person?.id === person);
 
+  type State =
+    | 'identity'
+    | 'livingsituation'
+    | 'money'
+    | 'health'
+    | 'checklist';
+  const [state, setState] = useState<State>('money');
```
Comment (tcmorris) [inline pages/applications/view/[id]/[person]/index.tsx:60]: Default to `identity`

Comment (tony-gledhill) [reply pages/applications/view/[id]/[person]/index.tsx]: This has been updated

File: pages/applications/view/[id]/[person]/index.tsx [hunk 2]
```diff
@@ -41,40 +51,91 @@ export default function ApplicationPersonPage({
...
+  const currentAccomodation = currentAccomodationCheckboxList(applicant);
+  const situation = situationCheckboxList(applicant);
+  const employment = employmentCheckboxList(applicant);
+  const incomeAndSavings = incomeAndSavingsCheckboxList(applicant);
+  const medicalDetails = medicalDetailsPageData(applicant);
+
   return (
     <UserContext.Provider value={{ user }}>
       <Layout>
-        <HeadingOne
```
Comment (tcmorris) [inline pages/applications/view/[id]/[person]/index.tsx:47]: This should be kept? we need a heading for the page.

**Review by tcmorris — APPROVED** [review]

File: pages/applications/view/[id]/[person]/index.tsx (hunk 2 above)
Comment (tony-gledhill) [reply pages/applications/view/[id]/[person]/index.tsx:47]: This has been re-added

File: pages/applications/view/[id]/[person]/index.tsx [hunk 3]
```diff
@@ -41,40 +51,99 @@ export default function ApplicationPersonPage({
...
       <Layout>
         <HeadingOne
-          content={
+           content={
             isMainApplicant
               ? 'Review main applicant'
               : 'Review household member'
           }
         />
-        <HeadingTwo
```
Comment (tcmorris) [inline pages/applications/view/[id]/[person]/index.tsx:54]: Applicant name is also in the designs.

**Pull request merged**